docs/README.md

```diff
@@ -78,6 +78,7 @@ macOS (OS X) development environment.
 
 ## Web based report management
 * [Webserver User Guide](web/user_guide.md)
+* [WEB GUI User Guide](web/server/www/userguide/userguide.md)
```

Comment:
This link will not work because it will search this file under `docs/web/server/www/userguide/userguide.md`. You have to use the following link: `/web/server/www/userguide/userguide.md`.